compute/admin_guide/install/deploy-defender/host/windows-host.adoc

```diff
@@ -29,7 +29,7 @@ h|
 h|Processes
 h|Network
 h|Filesystem
-h|CNNS
+h|CNNS <tagging-remove-for-SaaS-only>
```

Comment:
<tagging-remove-for-SaaS-only> remove this column and the respective rows for SaaS docs only.